### DISCUSSION TITLE
Missing binaries for v1.0.2

### DISCUSSION THREAD

**Discussion by marpada** [polarmutex/beancount-language-server#147] [General]
Hi,

First of all, thank you very much for this project !

I noticed that the v1.0.2 release lacks built binaries. This breaks some tools like [nvim-lsp-installer](https://github.com/williamboman/nvim-lsp-installer), plus raises the barrier for users not familiar with the build toolchain.

Would it be possible to add the binaries as in previous releases?

**Comment by polarmutex** [comment]
I will look into it, thanks for reporting it
